matrix/common/etl/transformers/specimen_library.py

```diff
@@ -91,13 +179,18 @@ def resolve_ontology_ols(self, term):
         try:
             ontology, term_id = term.split(":")
         except ValueError:
-            return None
+            try:
+                # Well this isn't right, but it's present in the metadata
+                ontology, term_id = term.split("_")
```

Comment:
i.e this is an upstream data sanitization problem?

Reply:
Yes it is. It is supposed to be fixed eventually, and in theory new validation tools should be preventing it now.

Reply:
Any way for us to know when this is fixed in prod data & we can remove?

Reply:
Apparently there will be a ticket in ingest-central: https://humancellatlas.slack.com/archives/C9ENBPVNW/p1572525618104200

Reply:
[Ticket](https://app.zenhub.com/workspaces/dcp-5ac7bcf9465cb172b77760d9/issues/humancellatlas/matrix-service/419). Resolving.